mne/channels/tests/test_dig_montage.py

```diff
@@ -0,0 +1,97 @@
+# Author: Teon Brooks <teon.brooks@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import inspect
+from nose.tools import assert_equal
+
+import numpy as np
+from numpy.testing import (assert_array_equal, assert_almost_equal,
+                           assert_allclose)
+
+from mne.channels.montage import _set_montage, read_dig_montage
+from mne.io.meas_info import _read_dig_points
+from mne.io.constants import FIFF
+from mne import create_info
+from mne.coreg import fit_matched_points
+from mne.io.kit import read_mrk
+from mne.transforms import apply_trans, get_ras_to_neuromag_trans
+
+
+FILE = inspect.getfile(inspect.currentframe())
+p_dir = op.dirname(op.abspath(FILE))
```

Comment:
eh? why not just `op.dirname(__file__)` like we do elsewhere? seems less painful.